python/lsst/rubintv/production/rubinTv.py

```diff
@@ -825,3 +850,201 @@ def beat(self, ensure=False, customFlatlinePeriod=None):
         if (elapsed >= self.uploadPeriod) or ensure or customFlatlinePeriod:
             if self.uploader.uploadHeartbeat(self.handle, forecast):  # returns True on successful upload
                 self.lastUpload = now  # only reset this if the upload was successful
+
+
+class CalibrateCcdRunner():
+    """Class for running CharacterizeImageTask and CalibrateTasks on images.
+
+    Runs these tasks and writes shards with various measured quantities for
+    upload to the table.
+    """
+    def __init__(self, outputRoot, *, doRaise=False, embargo=False):
+        self.dataProduct = 'quickLookExp'
+        self.uploader = Uploader()
+        self.butler = makeDefaultLatissButler(embargo=embargo, writeable=True)
+        self.log = _LOG.getChild("monitorChannel")
+        self.channel = 'auxtel_characterizeImage'
```

Comment:
1) This should probably be renamed to match the new class name.
2) If this ever actually uploads a plot itself (and it may well not) the this string will need adding to the list in `channels.py`.